virtual/secret-service/secret-service-0.ebuild

```diff
@@ -10,6 +10,5 @@ KEYWORDS="~alpha amd64 arm arm64 ~ia64 ~loong ~mips ppc ppc64 ~riscv sparc x86 ~
 RDEPEND="|| (
 	gnome-base/gnome-keyring
 	app-admin/keepassxc
+	>=kde-frameworks/kwallet-5.97
 )"
-# TODO: add the KDE provider here once ready, still WIP as of June 2021 though
-# (see https://bugs.kde.org/show_bug.cgi?id=313216)
```

Comment:
1. This would need a new revision.
2. The reason not to include it is that the linked bug remains open. See especially https://bugs.kde.org/show_bug.cgi?id=313216#c39...

Reply:
Other distributsion(afaik archlinux, fedora, opensuse tumbleweed) all have made it possible to switch over to the new implementation and the blocker is about removing `org.kde.kwalletd5` api from the kde frameworks and port everything to  QtKeyChain

Reply:
Okay. Have you tested it on Gentoo? See https://bugs.gentoo.org/880075#c1.

Reply:
yes it works without any issues

Reply:
I haven't figured out how to change the release for a virtual package if someone could point me in the right direction I will fix it

Reply:
I have bumped to version i think

Reply:
https://devmanual.gentoo.org/general-concepts/ebuild-revisions/ you rename the file with the revision you want it to be.

Reply:
ah then i have done it

Reply:
No, you add a new revision, bump it to EAPI=8, drop it to all `~arch` KEYWORDS, and keep the old one in place.